### DISCUSSION TITLE
I would like to have index.html returned in the urls

### DISCUSSION THREAD

**Discussion by tcrst** [thuliteio/doks#767] [Q&A]
I tried doing it through the uglyurls=true, but after enabling it the routing doesn't work anymore, any ideas?

**Comment by h-enk** [comment]
Can't reproduce your error /it's working for me with only setting `uglyurls = true` on a fresh install. Could you share your repo?

![Snag_165a9090](https://user-images.githubusercontent.com/3902872/172405216-96b944f0-a078-416f-bc02-ff50c0649d31.png)

**Reply by h-enk** [reply]
Note, see also https://discourse.gohugo.io/t/ugly-urls-with-page-resources/10037/5 + https://github.com/gohugoio/hugo/issues/4288
